purchases/src/main/kotlin/com/revenuecat/purchases/common/utils.kt

```diff
@@ -72,22 +72,23 @@ fun Locale.toBCP47(): String {
     return bcp47Tag.toString()
 }
 
+// Public because it's been used in the hybrids
```

Comment:
It's also exposed in the native right? (even though it shouldn't be used by users) So it actually should be public. Might be good to move to its own file...

Reply:
Moved it to its own class